frontend/src/features/properties/components/forms/subforms/InformationForm.tsx

```diff
@@ -39,6 +41,12 @@ const InformationForm: FunctionComponent<InformationFormProps> = (props: Informa
   const formikProps = useFormikContext();
   const { values } = useFormikContext<IProperty>();
   const projectNumber = getIn(values, withNameSpace('projectNumber'));
+  const keycloak = useKeycloakWrapper();
+
+  /** only SRES can change to Disposed  */
+  const classifications = keycloak.hasClaim(Claims.ADMIN_PROPERTIES)
+    ? props.classifications
+    : props.classifications.filter(x => x.label !== 'Disposed');
```

Comment:
changing this to use value and going to use the enum to check instead of hardcode